### ISSUE TITLE
Doesn't seem to be working on Windows

### ISSUE THREAD

**Issue by classicboss302** [AtomLinter/linter-rust#20]
It doesn't seem to be reporting issues anymore. It keeps reporting no issues, even though I deliberately made an issue that cargo warned about.

Versions it doesn't work with:
Linter 1.1.0
linter-rust 0.2.0
Rust 1.1

Versions it doesn't work with:
Linter 1.1.0
linter-rust 0.2.0
Rust 1.0

Versions it does work with:
Linter 1.1.0
linter-rust 0.1.0
Rust 1.0

**Comment by gmist** [comment]
In version 0.2.0 was a slight change in the method of finding executables ```rustc```/```cargo```. Please verify that the path of the Rustc home is correct.
![screen shot 2015-07-01 at 2 07 07 pm](https://cloud.githubusercontent.com/assets/47926/8453243/439623d8-1ffb-11e5-84ef-f2e4474102c3.png)

**Comment by classicboss302** [comment]
That helped a little bit. If there is an error, it shows that now. It still doesn't show warnings though.

**Comment by DontPanic345** [comment]
I have same issue, I have C:\Program Files\Rust stable 1.2 as the path

**Comment by gmist** [comment]
Try to change path to ```C:\Program Files\Rust stable 1.2\bin\rustc.exe```
![screen shot 2015-08-21 at 10 03 19 pm](https://cloud.githubusercontent.com/assets/47926/9416820/9ccd863a-4850-11e5-9423-ef23da7c2695.png)
